src/pl/plpython/Makefile

```diff
@@ -189,6 +191,7 @@ check: all submake
 	$(pg_regress_check) $(REGRESS_OPTS) $(REGRESS)
 
 installcheck: submake
+	cp expected/plpython_returns_2.out expected/plpython_returns.out
```

Comment:
I don't think this is needed since python3's `expected` is always in expected/**python3**/plpython_types.out